src/common/queries/auth/useGetGuestToken.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+import { ROUTE } from '@/common/constants/route';
+import { useNavigate } from 'react-router';
+import { getGuestToken } from '@/service/apis/auth';
+
+export const useGetGuestToken = () => {
+  const navigate = useNavigate();
+  return useQuery({
+    queryKey: ['guestToken'],
+    queryFn: async () => {
+      const response = await getGuestToken();
+      if (response?.accessToken) {
+        localStorage.setItem('accessToken', `Bearer ${response?.accessToken}`);
+        navigate(ROUTE.selectGroup);
+      } else {
+        console.error('Access token not found');
+      }
+    },
+    enabled: false, // refetch가 호출될 때만 실행되도록 설정
+  });
+};
```

Comment:
여기 queryFn에서 값을 반환하고 있지 않아서 에러가 발생하고 있습니다 🥹
```
Query data cannot be undefined. Please make sure to return a value other than undefined from your query function. Affected query key: ["guestToken"]
```
<img width="958" alt="Screen_Shot 2025-02-16 12 12 36" src="https://github.com/user-attachments/assets/3ddd2e21-c1e4-446e-8f71-f90679ac92fe" />

이렇게 반환값을 지정해주면 문제가 해결될 듯 합니다!

```suggestion
export const useGetGuestToken = () => {
  const navigate = useNavigate();
  return useQuery({
    queryKey: ['guestToken'],
    queryFn: async () => {
      const response = await getGuestToken();
      if (response?.accessToken) {
        localStorage.setItem('accessToken', `Bearer ${response?.accessToken}`);
        navigate(ROUTE.selectGroup);
        return response;
      }
      throw new Error('Access token not found'); // isError가 true가 되어 에러 처리 가능
    },
    enabled: false, // refetch가 호출될 때만 실행되도록 설정
  });
};
```

Reply:
앗 이런 에러가 있는지 몰랐네요... 바로 추가하겠습니다!